platform/darwin/src/NSExpression+MGLAdditions.mm

```diff
@@ -665,6 +704,26 @@ - (id)mgl_jsonExpressionObject {
                     [expressionObject addObject:obj.mgl_jsonExpressionObject];
                 }];
                 [expressionObject addObject:self.operand.mgl_jsonExpressionObject];
+                return expressionObject;
+            } else if ([function isEqualToString:@"mgl_matchWithOptions:default:"]) {
```

Comment:
@1ec5 do you think this new operand needs to be added as part of https://github.com/mapbox/mapbox-gl-native/pull/11472

Reply:
Yes, eventually everything here will need to be installed as aftermarket expressions. But we’d still need a selector-like function as backup, in case the approach in #11472 breaks for any reason.